docs/en/setup/advanced/AsyncEnhancement.md

```diff
@@ -0,0 +1,70 @@
+# Python Agent Asynchronous Enhancement
```

Comment:
This doc seems not to be linked from any place of the existing doc.
Do we need a new menu item?

Reply:
You mean I need to edit `docs/menu.yml` right?

It has been fixed.